spec/person_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../person'
+
+describe Person do
+  before do
+    @person = Person.new(20)
+  end
+
+  describe '#initialize' do
+    it 'initializes with an age' do
+      expect(@person.age).to eq(20)
+    end
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `of_age?, can_use_services?, correct_name and add_rental` class person method has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby
 def of_age?
    @age > 18
  end

  def can_use_services?
    is_of_age? || @parent_permission
  end

  def correct_name
    @name
  end

  def add_rental(rental)
    @rentals.push(rental) unless @rentals.include?(rental)
    rental.person = self
  end
```